tests/network/primary_pod_network.go

```diff
@@ -104,10 +104,15 @@ var _ = SIGDescribe("[Serial]Primary Pod Network", func() {
 					cleanupVMI(virtClient, vmi)
 				})
 
-				It("should report PodIP/s as its own on interface status", func() {
+				It("should report PodIP/s IPv4 as its own on interface status", func() {
 					vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
 					Eventually(vmiIP).Should(WithTransform(removeCIDR, Equal(vmiPod.Status.PodIP)), "should contain VMI Status IP as Pod status ip")
-					Eventually(vmiIPs).Should(ContainElements(vmi.Status.Interfaces[0].IP, libnet.DefaultIPv6Address), "should contain at IPs the IP and the cloud-init configured ipv6")
+					Eventually(vmiIPs).Should(WithTransform(removeCIDR, ContainElement(vmiPod.Status.PodIP)), "should contain IPv4 reported by guest agent")
+				})
+
+				It("should report VMIs static IPv6 at interface status", func() {
+					// Check first fixed IPv6 address so we are sure that GA has quick in
```

Comment:
Is this comment still relevant? You speak about checking "first", but I don't see any second check.

Reply:
Done